stylesheets/variables/_colors.styl

```diff
@@ -1,6 +1,7 @@
 $charles-blue     = #091f2f
 $freedom-red      = #fb4d42
-$optimistic-blue  = #288be4
+$optimistic-blue-light  = #288be4
+$optimistic-blue  = #1979cc
```

Comment:
Can you add comments about when to use the light one? Is it background colors and text on dark backgrounds?

Reply:
Totally can. 👍